src/kOS/Communication/kOSConnectivityParameters.cs

```diff
@@ -64,6 +64,8 @@ public override bool HasPresets
             }
         }
 
+        public override string DisplaySection => "kOS";
+
```

Comment:
This syntax was not understood when I tried to compile this in monodevelop.

I had to change it to the more explicit way of making a property:
```
        public override string DisplaySection { get {return "kOS";} }
```